attoparsec-iso8601/Data/Attoparsec/Time/Internal.hs

```diff
@@ -19,23 +19,15 @@ module Data.Attoparsec.Time.Internal
 
 import Prelude.Compat
 
+import Data.Fixed (Fixed(MkFixed), Pico)
 import Data.Int (Int64)
 import Data.Time
-import Unsafe.Coerce (unsafeCoerce)
-
-#if MIN_VERSION_time(1,6,0)
-
-import Data.Time.Clock (diffTimeToPicoseconds)
-
-#endif
-
-import Data.Fixed (Pico, Fixed(MkFixed))
 
 #if !MIN_VERSION_time(1,6,0)
+import Unsafe.Coerce (unsafeCoerce)
 
 diffTimeToPicoseconds :: DiffTime -> Integer
```

Comment:
This always exist through `time-compat` https://hackage.haskell.org/package/time-compat-1.9.3/docs/Data-Time-Clock-Compat.html#v:diffTimeToPicoseconds, i.e. import  `Data.Time.Clock.Compat` and we won't need CPP here.

Good catch.

Reply:
Great, I'll fix this too.